### DISCUSSION TITLE
Build requirements

### DISCUSSION THREAD

**Discussion by dgreenhouse1** [MathRobin/Never-Consent#73] [Q&A]
I'm working on my first enhancement for this project and have my own forked copy of the repo. Now I'd like to install my code in the browser for testing. Is anything other than executing build.sh required to get the extension built to the point of readiness to install on Firefox? I'm having some trouble with the result of the build and I'm sure I could figure it out with sufficient time but want to save troubleshooting time if someone already has the answer.

**Comment by MathRobin** [comment]
Hi ! Great to know that you are working on it !

But there is nothing more than build.sh currently. If you are looking into this script, there is everything needed to build locally.
Here https://github.com/MathRobin/Never-Consent/blob/master/build.sh#L22

I'm working on a V2 if you have any ideas to enhance the extension or the development process https://github.com/MathRobin/Never-Consent/projects/1
